data-raw/mario-kart-auction.R

```diff
@@ -0,0 +1,4 @@
+## code to prepare `mario-kart-auction` dataset goes here
+library(tidyverse)
+`mario-kart-auction` <- read_csv("https://www.openintro.org/data/csv/mariokart.csv")
```

Comment:
Please use snake case to name the dataset. It should not need backticks. Also pipe the URL to `read_csv`. Look at the processing for other datasets above as an example.